site/get-started.markdown

```diff
@@ -0,0 +1,142 @@
+---
+title: Get Started
+page: get-started
+isGettingStarted: true
+---
+
+# Get started
+Welcome, new Haskeller! Read on to quickly set up your Haskell dev environment, execute your first lines of code, and get directions for further learning!
+
+## Set up Haskell dev environment
+
+### GHCup - universal installer
+
+[GHCup](https://www.haskell.org/ghcup/#) is a universal installer for Haskell that will install for you everything you need to program in Haskell, and then will also help you manage those installations in the future (update, switch versions, ...).
+
+It is simple to use, works the same way on all platforms (Linux, macOS, Windows), and gives you one central place/method to take care of your Haskell development setup.
+
+Follow instructions at [GHCup webpage](https://www.haskell.org/ghcup/#) to install GHCup. Then, use it to install the Haskell Toolchain, which consists of:
+
+1. **GHC** -> Haskell compiler. We will use it below to run our examples, but in practice, you will mostly be using a build tool like `cabal` or `Stack` to build your code, instead of `GHC` directly.
+2. **HLS** -> Haskell Language Server -> You won't use this directly, instead your code editor will use it in the background to provide you with a great experience while editing Haskell code.
+3. **cabal** -> Haskell build tool -> You will use this to structure your Haskell projects, build them, run them, define dependencies, ...
+4. **Stack** -> Haskell build tool -> alternative to `cabal`
+
+
+<div class="bs-callout bs-callout-info">
+  <p>
+    <h4>cabal and Stack -> which one should I install?</h4>
+    cabal is the original build tool, while Stack was created as an alternative to cabal a time ago to solve some of the problems that cabal had and provide a more user-friendly experience. In the meantime, cabal solved most of those issues (including "cabal hell") and caught up with Stack regarding user experience, so the main difference between them at the moment is how they resolve dependencies, which for you as a beginner isn't a concern. Therefore, both are a good choice, and if not sure, you can install both and then use whatever the learning resources you will later use will point you to.
+  </p>
+</div>
+
+To confirm it is all installed correctly, you can run the following commands:
+```
+> ghc --version
+> haskell-language-server-wrapper --version
+> cabal --version                               # If you installed cabal
+> stack --version                               # If you installed Stack
+```
```

Comment:
I also removed this, I believe GHCup already gives enough feedback upon successful installation of tools.